googledatastore/README.md

```diff
@@ -20,6 +20,9 @@ LICENSE file.
 
 https://cloud.google.com/datastore/docs/concepts/overview?hl=en
 
+Please refer to https://cloud.google.com/datastore/docs/apis/overview for more information on
```

Comment:
The old link is an overview about Google Cloud Datastore. The new link contains only v1beta3 API documentation.